src/script/components/UserList/components/UserListItem/UserListItem.tsx

```diff
@@ -225,65 +180,36 @@ const ParticipantItem = <UserType extends User | ServiceEntity>(
               </span>
             )}
 
-            {isUser && isSelfVerified && isVerified && (
+            {isSelfVerified && isVerified && (
               <span className="verified-icon">
                 <Icon.Verified data-uie-name="status-verified" />
               </span>
             )}
-
-            {callParticipant && (
-              <>
-                {sharesScreen && <Icon.Screenshare className="screenshare-icon" data-uie-name="status-screenshare" />}
-
-                {sharesCamera && <Icon.Camera className="camera-icon" data-uie-name="status-video" />}
-
-                {!isMuted && (
-                  <ParticipantMicOnIcon
-                    className="participant-mic-on-icon"
-                    isActive={isActivelySpeaking}
-                    data-uie-name={isActivelySpeaking ? 'status-active-speaking' : 'status-audio-on'}
-                  />
-                )}
-
-                {isMuted && (
-                  <Icon.MicOff
-                    className="mic-off-icon"
-                    data-uie-name="status-audio-off"
-                    style={{height: 12, width: 12}}
-                  />
-                )}
-              </>
-            )}
           </>
         )}
       </InViewport>
     );
   };
 
   const dataUieValues = {
-    'data-uie-name': isUser ? 'item-user' : 'item-service',
-    'data-uie-value': participantName,
+    'data-uie-name': 'item-user',
+    'data-uie-value': userName,
   };
 
   const commonClassName = cx('participant-item-wrapper', {
-    highlighted,
```

Comment:
we should not remove this one, it's `highlighted` classname that won't be added now. Should be `highlighted: isHighlighted` now 👌